<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMTP-клиент для отправки почтовых сообщений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>Рабочая панель</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MSTP-Server:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverField">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>smtp.gmail.com: 587</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправитель:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SenderField">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordFile">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Получатель:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recipientField">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Копии:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="copiesList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Тема:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subjectField">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="messageField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>351</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>590</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>590</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Сообщение:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>461</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>Административная панель</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Группы рассылок:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="groupButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>340</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить Группу</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adressField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="adressButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>380</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить Адресс</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>441</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
